docs/package.json

```diff
@@ -9,7 +9,7 @@
   },
   "devDependencies": {
     "@shikijs/vitepress-twoslash": "^1.22.1",
-    "@types/express": "^4.17.21",
+    "@types/express": "^5.0.0",
```

Comment:
This will make the twoslash type hover to use the types from express@5, but the content in the hover was same for our case.

Reply:
Ah you're right. Se we can't bump this still until express v5 is marked stable. I'll revert this part back then